index.ts

```diff
@@ -72,18 +102,33 @@ class WalletClient {
       background: 'white',
       borderRadius: '16px',
       padding: '28px',
+      paddingTop: '47px',
       textAlign: 'center',
       minWidth: '300px',
-      fontFamily: 'system-ui, -apple-system, sans-serif',
+      fontFamily: 'Sora',
+      display: 'flex',
+      justifyContent: 'center',
+      flexDirection: 'column',
+      alignItems: 'center',
+      backgroundRepeat: 'no-repeat',
     });
 
+    content.style.backgroundImage = `url(${
+      'data:image/svg+xml;base64,' + incodedPattern
+    })`;
+
     content.innerHTML = `
```

Comment:
please think about deviding html templates to other file, also create folder for html templates